Scripts/datahandling/zonedata.py

```diff
@@ -100,6 +100,8 @@ def __init__(self, data_dir: Path, zone_numbers: Sequence,
         self["outside_municipality"] = ~within_municipality
         for dummy in ("Helsingin_kantakaupunki", "Tampereen_kantakaupunki"):
             self[dummy] = self.dummy("subarea", dummy)
+        for key in data["aggregate_results_submodel"].unique():
+            self["population_" + key] = (data["aggregate_results_submodel"] == key) * pop
```

Comment:
This could also be written as:

```suggestion
            self["population_" + key] = self.dummy("submodel", key) * pop
```